template/pyproject.toml.jinja

```diff
@@ -63,9 +60,15 @@ docs = [
 homepage = "https://docs.{{packagename}}.org"
 documentation = "https://docs.{{packagename}}.org"
 
+[tool.hatch.version]
```

Comment:
It was attempted to get the version nr from the git tag using:
`setuptools-git-versioning`

However I have not yet been able to get that functionality to work.

Reply:
Also, see comment below